src/ucp/core/ucp_context.c

```diff
@@ -141,6 +141,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "Maximal number of devices on which a rendezvous operation may be executed in parallel",
    ucs_offsetof(ucp_config_t, ctx.max_rndv_lanes), UCS_CONFIG_TYPE_UINT},
 
+   {"RNDV_BALANCING_ENABLE", "y",
```

Comment:
It is not very clear what the balancing actually means.  I would suggest to elaborate what do you mean. It is pretty clear what it is doing from the code but I don't expect our user to dig through the code.

Reply:
@yosefe - maybe we should require FAQ entry for each new tuning option ... anywhere